config/locales/views/mailer/hi.yml

```diff
@@ -0,0 +1,39 @@
+  group_mailer:
+    group_created_html: "आपने सफलतापूर्वक एक नया समूह बना लिया है, <span class='mailer-highlighted-text'> %{name} </span>।"
+    view_group: "समूह देखें"
+    member_added_to_group_html: "आपको <span class='mailer-highlighted-text'>%{group_name}</span> नाम के एक नए समूह में जोड़ा गया है। <br>
+    समूह का मार्गदर्शन <span class='mailer-highlighted-text'>%{user_name}</span> द्वारा किया जाता है।"
+  assignment_mailer:
+    assignment_created_html: "<span class='mailer-highlighted-text'>%{name} (%{deadline})</span> नाम का एक की नयी असाइनमेंट <span class='mailer-highlighted-text'> %{group_name} में असाइन की की गयी है </span>. <br>
+    असाइनमेंट का मार्गदर्शन <span class='mailer-highlighted-text'>%{mentor_name}</span> द्वारा किया जाता है"
+    view_assignment: "असाइनमेंट देखें"
+    assignment_updated_html: "असाइनमेंट <span class='mailer-highlighted-text'>%{name} (%{deadline})</span> को अपडेट कर दिया गया है। <br>
+    असाइनमेंट का मार्गदर्शन <span class='mailer-highlighted-text'>%{mentor_name}</span> द्वारा किया जाता है"
+  user_mailer:
+    custom_email:
+      hello_user: "नमस्ते, %{name}"
+    featured_circuit_email:
+      circuit_featured_html: "वाह! आपकी परियोजना, <span class='mailer-highlighted-text'>%{name}</span> मंच पर प्रदर्शित कि गयी है।"
+    forked_project_email:
+      project_forked_html: "आपने <span class='mailer-highlighted-text'>%{original_author}</span> से एक नयी परियोजना सफलतापूर्वक फोर्क कि है। <br>
+      आपके फोर्क किए गए परियोजना का नाम <span class='mailer-highlighted-text'>%{forked_name}</span> है और परियोजना का एक्सेस टाइप <span class='mailer-highlighted-text'>%{access_type}</span > है।"
+    new_project_email:
+      project_created_html: "आपने सफलतापूर्वक नयी परियोजना बना लि है, <span class='mailer-highlighted-text'>%{name}</span>।"
+    welcome_email:
+      know_more: "अधिक जानने के लिए"
+      welcome_text_html: "साइन अप करने के लिए धन्यवाद।<br> सर्किटवर्स के साथ आप साधारण गेट से लेकर काम्प्लेक्स सेक्वेंशिअल सर्किट्स, प्लॉट टाइमिंग डायग्राम, आटोमेटिक सर्किट जनरेशन, स्टैण्डर्ड आईसी के बारे में अधिक, और बहुत कुछ जानिए, आप सर्किटवर्स के साथ मुफ्त में डिजिटल लॉजिक सर्किट डिजाइन और बहुत कुछ कर सकते हैं!"
+      design_heading: "डिज़ाइन करना"
+      design_description: "क्या आपको पता नहीं कैसे डिजिटल सर्किट डिजाइन करना है? सर्किटवर्स, शुरुआती लोगों के लिए उपयोग करना आसान है। सर्किटवर्स डॉक्यूमेंटेशन आपको जल्दी से शुरू करने में मदद करेगा।"
+      learn_heading: "सीखना"
+      learn_description: "क्या आप डिजिटल सर्किट के बारे में अधिक जानना चाहते हैं? सर्किटवर्स इंटरैक्टिव बुक आपको डिजिटल सर्किट कैसे काम करती है, इसकी पूरी समझ देती है।"
+  pending_invitation_email:
+    main_heading: "सर्किटवर्स में आपका स्वागत है"
+    main_description_html: "आपको %{user_name} द्वारा %{name} नाम के एक नए समूह में जोड़ा गया है।<br><br>
+     डिजिटल सर्किट की दुनिया को एक्स्प्लोर करने के लिए सर्किटवर्स के साथ साइन अप करें। एक बार जब आप सर्किटवर्स के साथ साइन अप कर लेते हैं तो आप समूह का उपयोग कर सकते हैं।"
```

Comment:
Maybe rephrase this to - digital circuit ki duniya mei khoj karne ke liye” keeping rest of the sentence same